src/app/services/tests/AuthenticationService.spec.ts

```diff
@@ -1,49 +1,83 @@
-import { AuthenticationService, AuthUser } from '../AuthenticationService';
-import { AngularFireAuth } from '@angular/fire/auth';
-import { AngularFirestore } from '@angular/fire/firestore';
-import { Observable, of } from 'rxjs';
-import { fakeAsync, tick } from '@angular/core/testing';
-import { delay } from 'rxjs/operators';
+import { AuthenticationService, AuthUser, RTDB } from '../AuthenticationService';
+import { Observable, ReplaySubject, Subscription } from 'rxjs';
+import { fakeAsync, TestBed } from '@angular/core/testing';
 import firebase from 'firebase/app';
 import 'firebase/auth';
 import { Injectable } from '@angular/core';
+import { MGPValidation } from 'src/app/utils/MGPValidation';
+import { MGPFallible } from 'src/app/utils/MGPFallible';
+import { Utils } from 'src/app/utils/utils';
+import { UserDAO } from 'src/app/dao/UserDAO';
+import { setupEmulators } from 'src/app/utils/tests/TestUtils.spec';
+
+class RTDBSpec {
+    // TODO: these are stubs that can be removed after the RTDB functions ticket has been done
```

Comment:
so, do you ticket this or do you do this ?

Reply:
ticketté

Reply:
Ah yes, I see the ticket, you can remove the comment then !

Reply:
done